main/src/main/java/cgeo/geocaching/connector/gc/GCParser.java

```diff
@@ -18,6 +18,8 @@
 import cgeo.geocaching.log.LogEntry;
 import cgeo.geocaching.log.LogType;
 import cgeo.geocaching.log.LogTypeTrackable;
+import cgeo.geocaching.log.LogUtils;
```

Comment:
You placed the logic into GCParser. Wouldn't that mean that it won't work for eg OC caches? Or are those not affected by the bug?

Reply:
Good question, will check on this

Reply:
Seems that from OC we receive already the correct time, so need to merge the time stamp

Reply:
Don't know if other connectors support the "friend", which is also considered here